featuretools/__init__.py

```diff
@@ -35,6 +35,7 @@
     try:
         sys.modules["featuretools." + entry_point.name] = entry_point.load()
     except Exception:
-        message = "Featuretools failed to load plugin {} from library {}"
+        message = "Featuretools failed to load plugin {} from library {}. "
+        message += "For more information, set logging to debug."
```

Comment:
Let's change the beginning of the second line to "For a full stack trace, . . ."